go/fix-chain/main/fix-chain.go

```diff
@@ -0,0 +1,116 @@
+package main
```

Comment:
Right now, our repo is a little messy for `go get` to work nicely, but in an ideal future, you could run `go get github.com/google/certificate-transparency`, and immediately have all the Go-based bits installed and ready to go.

Yeah, I know `preload` does that too... :stuck_out_tongue: